### PR TITLE
feat: write collected diff files to external repository

### PR DESCRIPTION
**Pull request by javierron**
Functionality to write diff files containing one-line-fixes into an external repository.

Currently using [repairnator/continuous-learning-data](https://github.com/repairnator/continuous-learning-data)

### CODE REVIEW

File: repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/SequencerLearnerScanner.java [hunk 1]
```diff
@@ -26,7 +27,14 @@ public static void main(String[] args) {
     public void run() {
         LOGGER.debug("Start running inspect Jobs...");
         JobHelperv2 jobHelperv2 = new JobHelperv2(RepairnatorConfig.getInstance().getJTravis());
-        SequencerCollector collector = new SequencerCollector();
+        SequencerCollector collector = null;
+        
+        try {
+            collector = new SequencerCollector();
+        } catch (IOException e1) {
+            // TODO Auto-generated catch block
```
Comment (monperrus) [inline repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/SequencerLearnerScanner.java]: avoid TODO, avoid printStackTrace, use `throw new RuntimeException(e)` instead

**Comment by monperrus** [comment]
Thanks for the PR. Could you add a test case to specify this new feature?

**Comment by javierron** [comment]
> Thanks for the PR. Could you add a test case to specify this new feature?

Test case is now in place.

**Comment by monperrus** [comment]
Activated CI on PR again. We have a failure to handle, partly due to #744. Fixing it now with #983.

**Comment by monperrus** [comment]
CI Fixed. This PR introduces a regression, see https://travis-ci.org/eclipse/repairnator/jobs/621918221

```
  TestRTScanner.testDockerPipelineRunner:41 » NoSuchMethod org.apache.http.impl....
  TestSequencerCollector.initMocks:38 Mockito 
Cannot instantiate @InjectMocks f...
```

This is likely due to the introduction of the new library which changes some versions in the underlying dependencies. Could you have a look?

**Comment by javierron** [comment]
OK. It is fixed now, there were 2 problems.
 1. JGit package version created a dependency conflict.
 1. Some local paths were not resolving when running the new test case.

**Comment by monperrus** [comment]
Thanks a lot!

**Pull request merged**